<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlopeWidget</class>
 <widget class="QWidget" name="SlopeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>206</width>
    <height>126</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QComboBox" name="slopeValComboBox">
     <item>
      <property name="text">
       <string>Degrees</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Percentage</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>29</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="vunitComboBox">
     <item>
      <property name="text">
       <string>Meters</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>US Feet</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Slope value:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Algorithm:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Vertical unit:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QComboBox" name="algComboBox">
     <item>
      <property name="text">
       <string>ZevenbergenThorne</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Horn</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
